gog/graph.go

```diff
@@ -274,5 +272,8 @@ func (g *Grapher) makeDefInfo(obj types.Object) (*DefKey, *defInfo) {
 		path = append([]string{filepath.Base(p.Filename)}, path...)
 	}
 
-	return &DefKey{obj.Pkg().Path(), path}, &defInfo{pkgscope: g.pkgscope[obj], exported: g.exported[obj]}
+	return &DefKey{obj.Pkg().Path(), path}, &defInfo{
+		pkgscope: g.pkgscope[obj],
+		exported: obj.Exported() && (obj.Parent() == nil || obj.Parent() == obj.Pkg().Scope()),
```

Comment:
Problem is that I'm not exactly sure what the semantics are, which makes it hard to find a good function name. Right now I'm trying to refactor without changing the output and this worked. Later I may want to investigate how the output is actually used.